scripts/generate_landings_risk_report.py

```diff
@@ -895,7 +899,9 @@ def go(self, days: int) -> None:
             ):
                 bug_map[bug["id"]] = bug
 
-        self.generate_landings_by_date(bug_map, bugs, self.get_blocking_of(meta_bugs))
+        self.generate_landings_by_date(
+            bug_map, bugs, self.get_blocking_of(bug_map, meta_bugs)
```

Comment:
As a follow-up, what do you think about adding bug_map as a cached property (i.e., `@cached_property`)? In that way, we do not need to pass it with the function calls.